### DISCUSSION TITLE
WTF Academy 2022总结 和 2023展望

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#395] [General]
总结2022年
1. Web3 开源教程: WTF Solidity, WTF Ethers, WTF HTML/CSS/JavaScript, WTF gm（Web3名词）, WTF Cairo（StarkNet）
2. 测试和技能认证网站: wtf.academy
3. 获取资助
4. 建立社区
5. 贡献者参与共建

展望2023年
1. 继续更新开源教程（50%）
 - WTF Solidity: Degen篇，PR模版
 - WTF Ethers: 工具篇（Hardhat + Ethers, wagmi, rainbowkit, ?Alchemy API）
 - WTF Cairo: Cairo 1.0
 - WTF JavaScript
 - WTF Zero-Knowledge Proof
 - WTF Graph
 - WTF Foundry
 - ...
2. 社区项目/合作（20%）
 - WTF Academy网站：多语言，更多课程，在线编译/检测（Alchemy University），个人中心。
 - StarkWare: WTF Academy SnarkNet，让开发者熟悉在StarkNet上开发。
 - Scroll: 让开发者在Scroll上部署合约，领取NFT凭证。
 - Ether.js v5.4 文档翻译
 - WTF Academy x PKU Blockchain 黑客松（公平，不允许2022年已有项目参与）
 - 媒体关系
 - 每季度会做一个社区项目（公共品），解决散户难题。
 - ...
 3. 外部合作（10%）
  - 提供: 教程，培训，测试，认证。
  - 需求: 社区/贡献者/学员获得利益。
 4. 社区运营/治理（20%）
  - 学员和贡献者身份上链（SBT），对提案有投票权（测试阶段）。
  - WTF Academy Share 技术分享会（2周一次，限制在1小时），给分享者提供SBT。
  - 社区任务同步到 dework 平台，激励制度参考Code4rena
  - 2022年贡献者回溯空投（$5000）
   -- 贡献评级依据：PR，Commit，任务，教程，社区贡献等。
   -- 贡献者分5档：Founding, Degen, Pro, Advanced, Intro
   -- 每档分到20%奖励，在这一档的贡献者中评分。
   -- 上一档的贡献者也会计入下一档奖励中。
   -- 单人最多分总奖池的20%（限制0xAA）。
 - 思考 WTF Academy 的 Labs 制度草稿。
 - 是否要开启融资？

**Comment by warden666x** [comment]
加油，希望社区越来越好！
